utils/requests.ts

```diff
@@ -1,10 +1,10 @@
+import axios from "axios";
 import * as mongoose from "mongoose";
-import {updateModel, userModel} from "../models/models";
+import { getSession } from "next-auth/react";
 import short from "short-uuid";
-import axios from "axios";
-import {getSession} from "next-auth/react";
-import { SortBy, Update, User } from "./types";
+import { updateModel, userModel } from "../models/models";
 import getLookup from "./getLookup";
+import { SortBy, Update, User } from "./types";
 
 export interface GetUpdateRequestResponse {user: User, update: Update & {mentionedUsersArr: User[]}};
```

Comment:
lowk unrelated to this pr but this file is kinda gross lol why is the api logic not in the api file 😭

Reply:
ya most of the older updately apis are structured like that. with api logic in `requests.ts` then the api file calling functions from it. this seems like smtn we can refactor if we ever want to do a big updately refactor using samson's new react learnings :sparkles:

Reply:
wait actually i think that could be fun. i think id learn a lot trying to do all of updately more efficiently & w better engineering practices

Reply:
esp if i had dron's guidance too